### DISCUSSION TITLE
Feature Request: Playnite Extension

### DISCUSSION THREAD

**Discussion by GCTuba** [Raicuparta/rai-pal#115] [Announcements]
It would be helpful if an extension was made in Playnite that could tag every game in the library that is compatible with UEVR, the status of implementation, and download control profiles. Very similar to the current implementation but it would work for every library and not just Steam.

**Comment by Raicuparta** [comment]
Good point! The database that Rai Pal and the UEVR DB website will use for compatibility scores + profiles will be open for anyone to use, so there's nothing stopping anyone from making such a plugin for Playnite.

But I wouldn't say that's really relevant to Rai Pal? So maybe not the right place to open this issue. I'll move this to discussions.

**Comment by GCTuba** [comment]
I considered opening it in UEVR github but it didn't seem relevant there. There's not one individual who creates Playnite extensions so I'm not sure where else to open it.
